### ISSUE TITLE
Callling /control/parental/enable failing with `only content-type application/json is allowed`

### ISSUE THREAD

**Issue by hastarin** [frenck/python-adguardhome#795]
# Problem/Motivation

I'm using the AdGuard Home addon in Home Assistant and believe I've traced a bug I'm having to the code in this repository. 

Specifically, the call to /control/parental/enable is failing.

## Expected behavior

Should be able to enable the parental filtering via the API wrapper.

## Actual behavior

Throws an error with: `only content-type application/json is allowed`.

## Steps to reproduce

- pip install adguardhome
- Temporarily configure AdGuard Home Addon in HA to `leave_front_door_open` and set a port (eg 3001) for the Web interface
![image](https://github.com/frenck/python-adguardhome/assets/452393/66604f02-1a11-4179-92be-4001c506fefd)
- Edit the following and change the IP/port to suit your AdGuard Home installation
```python
"""Asynchronous Python client for the AdGuard Home API."""

import asyncio

from adguardhome import AdGuardHome


async def main():
    """Show example how to get status of your AdGuard Home instance."""
    async with AdGuardHome(host="192.168.1.2", port="3001", tls=True, verify_ssl=False) as adguard:
        version = await adguard.version()
        print("AdGuard version:", version)

        await adguard.parental.disable()

        active = await adguard.parental.enabled()
        active = "Yes" if active else "No"
        print("Parental control enabled?", active)

        await adguard.parental.enable()
        active = await adguard.parental.enabled()
        active = "Yes" if active else "No"
        print("Parental control enabled?", active)


if __name__ == "__main__":
    loop = asyncio.get_event_loop()
    loop.run_until_complete(main())
```
- python3 adguard_test.py

## Proposed changes

It appears the AdGuard Home API now enforces a "Content-Type: application/json" for the API when a body is present. The request to /control/parental/enable is adding a body but it's not JSON.

From what I can tell from https://github.com/AdguardTeam/AdGuardHome/blob/master/openapi/openapi.yaml the endpoint doesn't actually expect a body.

The fix is to simply remove the data argument on line 36 of parental.py.

**Comment by hastarin** [comment]
This is not stale. Frenck is travelling and unable to action it.

**Comment by stamandr** [comment]
Many of us have been waiting a long time for this fix....

**Comment by frenck** [comment]
fixed in #796
